routers/api/v1/api.go

```diff
@@ -807,7 +807,11 @@ func RegisterRoutes(m *macaron.Macaron) {
 									Get(repo.GetPullReviewComments)
 							})
 						})
-
+						m.Group("/requested_reviewers", func() {
+							m.Combo("").
+								Delete(reqToken(), bind(api.PullReviewRequestOptions{}), repo.DeletReviewRequests).
+								Post(reqToken(), bind(api.PullReviewRequestOptions{}), repo.CreatReviewRequests)
+						})
```

Comment:
I think you can just go with `m.Combo("/requested_reviewers")` instead of `m.Group`.